modules/functional-test/src/main/java/com/intuit/wasabi/tests/perftest/PerfTest.java

```diff
@@ -0,0 +1,127 @@
+
+            // pause the experiment
+            exp.state = Constants.EXPERIMENT_STATE_PAUSED;
+            putExperiment(exp);
+
+            // terminate the experiment
+            exp.state = Constants.EXPERIMENT_STATE_TERMINATED;
+            putExperiment(exp);
+
+            // delete the experiment
+            deleteExperiment(exp);
+        }
+
+        List<Experiment> experimentsListOfQBO = getApplicationExperiments(qboApplication);
+
+        for (Experiment exp : experimentsListOfQBO) {
+
+            // pause the experiment
+            exp.state = Constants.EXPERIMENT_STATE_PAUSED;
+            putExperiment(exp);
+
+            // terminate the experiment
+            exp.state = Constants.EXPERIMENT_STATE_TERMINATED;
+            putExperiment(exp);
+
+            // delete the experiment
+            deleteExperiment(exp);
+        }
+
+    }
+
+    @Test
+    public void prepareData() {
+
+        // now lets create experiments for each applications
+        for (int i = 1; i <= 15; i++) {
+            Experiment exp = ExperimentFactory.createExperiment("wasabi-perftest-ctgapp-exp" + i)
+                    .setApplication(ctgApplication);
+            experimentListOfCTG.add(exp);
+        }
+
+        experimentListOfCTG = postExperiments(experimentListOfCTG);
+
+        // now lets create experiments for each applications
+        for (int i = 1; i <= 15; i++) {
+            Experiment exp = ExperimentFactory.createExperiment("wasabi-perftest-qboapp-exp" + i)
+                    .setApplication(qboApplication);
+            experimentListOfQBO.add(exp);
+        }
+
+        // lets create experiment aaa that is used for single assignment
+        Experiment exp = ExperimentFactory.createExperiment("aaa").setApplication(qboApplication);
+        experimentListOfQBO.add(exp);
+        experimentListOfQBO = postExperiments(experimentListOfQBO);
+
+        for (int i = 1; i <= 15; i++) {
+            List<Bucket> bucketList = BucketFactory.createBuckets(experimentListOfCTG.get(i - 1), 5);
+            postBuckets(bucketList);
+        }
+
+        for (int i = 1; i <= 16; i++) {
+            List<Bucket> bucketList = BucketFactory.createBuckets(experimentListOfQBO.get(i - 1), 5);
+            postBuckets(bucketList);
+        }
+
+        // lets change state of the experiment to running
+        for (int i = 1; i <= 15; i++) {
+            Experiment experiment = experimentListOfCTG.get(i - 1);
+            experiment.state = Constants.EXPERIMENT_STATE_RUNNING;
+            putExperiment(experiment);
+        }
+
+        // lets change state of the experiment to running
+        for (int i = 1; i <= 16; i++) {
+            Experiment experiment = experimentListOfQBO.get(i - 1);
+            experiment.state = Constants.EXPERIMENT_STATE_RUNNING;
+            putExperiment(experiment);
+        }
+        
+        System.out.println("experiment ID:" + experimentListOfQBO.get(0).id);
```

Comment:
souts are never a good idea to get information from the program. Suggest moving it to a logger.